pkg/virt-controller/watch/vmi.go

```diff
@@ -360,22 +360,6 @@ func (c *VMIController) updateStatus(vmi *virtv1.VirtualMachineInstance, pod *k8
 					c.recorder.Eventf(vmi, k8sv1.EventTypeWarning, FailedGuaranteePodResourcesReason, "failed to guarantee pod resources")
 					syncErr = &syncErrorImpl{fmt.Errorf("failed to guarantee pod resources"), FailedGuaranteePodResourcesReason}
 				} else {
-
-					// vmi is still owned by the controller but pod is already ready,
```

Comment:
The comment is still relevant and usefull. Please don't remove it.

Reply:
reverted now